src/main/java/com/fdmgroup/forex/exceptions/InsufficientFundsException.java

```diff
@@ -0,0 +1,9 @@
+package com.fdmgroup.forex.exceptions;
+
```

Comment:
@ResponseStatus(HttpStatus.FORBIDDEN)

Reply:
Resolved in [4a1c1eb](https://github.com/forex-agile/backend/pull/2/commits/4a1c1ebe5c65cda071ffc01d4c60acce08a7a621)

Reply:
Resolved by commit 4a1c1eb.